_data/committee/2023-24.yaml

```diff
@@ -169,7 +169,7 @@ people:
 
   - name: Rohit J.
     pronouns: He/Him
-    discordtag: bluka479
+    discordtag: ''
```

Comment:
You'd want to remove this line completely instead

Reply:
Was there a particular reason for this? Was initially worried it would break it altogether and just needed a blank input.

Reply:
I believe giving it a blank input would leave the option there, just with nothing in it, but removing it entirely removes the element from the page rather than having it blank 

(this could be absolute waffle but I think that's how it works)

Reply:
Templates can check if a variable is set like so: `{% if profile.discordtag %}`, which can then control if the Discord tag section is rendered. If it's set to an empty string, then that check is truthy and the section would be rendered with an empty string as opposed to not being rendered at all.

I'm not sure if that's the case in this instance, but I do know that a Discord tag is not required and it may as well be completely removed instead of simply emptied.